app/models/prePopulation/EmploymentPrePopulationResponse.scala

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.prePopulation
+
+import models.prePopulation.EmploymentPrePopulationResponse.EmploymentRadioPrePop
+import play.api.libs.json.{Json, Reads}
+
+case class EmploymentPrePopulationResponse(hasEmployment: Boolean) {
+  val hasPrePop: Boolean = hasEmployment
+
+  def toPrePopRadioModel: EmploymentRadioPrePop = new EmploymentRadioPrePop(hasPrePop)
+
+}
+
+object EmploymentPrePopulationResponse {
+  implicit val reads: Reads[EmploymentPrePopulationResponse] = Json.reads[EmploymentPrePopulationResponse]
+
+  val empty: EmploymentPrePopulationResponse = EmploymentPrePopulationResponse(hasEmployment = false)
+
+  class EmploymentRadioPrePop(hasEmployment: Boolean) extends EmploymentPrePopulationResponse(hasEmployment)
+    with PrePopulationResponse[Boolean] {
+
+    override def toPageModel: Boolean = hasEmployment
+
+    override def toMessageString(isAgent: Boolean): String = ""
+  }
+
+  object EmploymentRadioPrePop {
+    val empty: EmploymentRadioPrePop = new EmploymentRadioPrePop(hasEmployment = false)
+  }
+}
```

Comment:
Just a minor query, why has this file been formatted like this as opposed to the format from CIS?